src/action_builder.js

```diff
@@ -936,10 +955,19 @@ module.exports = class ActionBuilder {
 
     await this.updateLinks();
 
+    if (this._gateways.fastly && this._gateways.fastly.ready()) {
+      Object.values(this._deployers).forEach((d) => {
+        this._gateways.fastly.withDeployer(d);
+      });
+
+      this._gateways.fastly.init();
+      await this._gateways.fastly.deploy();
```

Comment:
@trieloff shouldn't we put this behind another _command_ ? like `--update-gateway` 
I find the precense of the gateway a weak trigger, especially if the env vars are always there in a ci.

Reply:
Good idea